.azure/export.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
## Arc IaC Automation generated
Next steps:
 - Wait for the export workflow run
 - If the run failed, check the diff file.
 - If succeed, resolve this comment and merge the pull request

If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com